js/gallery.js

```diff
@@ -1 +1,19 @@
-// Gallery page logic should be placed here.
\ No newline at end of file
+// Gallery page logic should be placed here.
+/*charge the images with backgroundImage */
+window.onload = function chargeImages()
+{
+const images = [
```

Comment:
Is not `Images` too generic as name for an object? In this case.

Reply:
I changed the name of this function  for "imagesGallery"